### PR TITLE
Create units_README.md

### LINKED ISSUE #493: README for pint unit registry generator
Create a README file for documentation of the pint unit registry generator (which imports the QUDT unit ontology).

https://github.com/SINTEF/dlite/blob/master/bindings/python/triplestore/units.py

### PR DESCRIPTION
**Pull request by ajeklund**
# Description:

Adds documentation for the Pint unit registry generator (based on the QUDT ontology).

Closes #493.

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [x] Documentation update.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

File: bindings/python/triplestore/units_README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Introduction and usage
+----------------------
+
+The units.py file contains the get_pint_registry() function, which downloads 
+the [QUDT UNITS](https://www.qudt.org/doc/DOC_VOCAB-UNITS.html) vocabulary 
+and uses its contents to generate a unit registry file for 
+[Pint](https://pint.readthedocs.io). The function then uses the generated 
+registry file to generate and return a Pint UnitRegistry object.
+
+The unit registry file is cached, and the default behavior is to not
```
Comment (jesper-friis) [inline bindings/python/triplestore/units_README.md:13]: Where is it cached?

**Review by jesper-friis — APPROVED** [review]
Looks fine. I approve, but please take a look at the comment. 

I think a better place for this unit handling would be as a sub-package in tripper rather than in DLite. But that would be a separate issue.

**Comment by ajeklund** [comment]
Discovered that there is an error when trying to generate a unit registry file using Windows. Reported as issue #497 and mentioned in this readme.

**Review by jesper-friis — APPROVED** [review]
Nice. Approved

**Pull request merged**